### DISCUSSION TITLE
Osu Lazer Mania, mobile input idea.

### DISCUSSION THREAD

**Discussion by PringleKitten** [ppy/osu#26380] [Ideas]
So, you changed it so the skin is massive to fit the playing field and make it easier, but now no skin can easily be read unless it is a bar.
My idea is instead, bring back the normal play field ui, and instead add tiles on the screen that can be touched to play. Reference from FNF Mobile.

**Comment by Rekunan** [comment]
By FNF mobile do you mean this? (from https://www.youtube.com/watch?v=hV4fzdQRAkY)
![image](https://github.com/ppy/osu/assets/24501632/3630b317-b54b-4d60-bbae-9564f45c1f46)

**Reply by PringleKitten** [reply]
No the way REAL fnf mods do it off their apk files. Not fake ones in the play store

**Reply by Rekunan** [reply]
Would it be possible if you could send a screenshot of what you're talking about then? I seem to be coming across conflicting information.

**Reply by PringleKitten** [reply]
https://youtu.be/u41lUSn-wIg?si=KWVIQOG6u2Bdix0p

Here is what I mean

**Reply by Rekunan** [reply]
Ahh, ok, so your idea is to split the screen into 4 equally sized segments which are then respectively mapped for input.

That does sound interesting, tho I wonder how that'll turn out with higher keycounts. I'm guessing still a massive benefit, but the benefit decreases marginally as you increase the keycount.

**Reply by PringleKitten** [reply]
Still better than the input osu is using rn where you click the small lane of icons, especially since you guys broke every skin, you can bring back how it was before to fix all skins and use the tile input, tho I think its better to just add an option on how people wanna play in options:

New Option in Input:
"Play Area Style"(or something idk):
-Tiles(The skin isnt stretched across the screen like it is now, and Doesn't mean you HAVE to use the tiles, you can also chsnge the opactiy from 0-1 so if you want them invisible you can do that, keyboard and stuff would still work as well if you wanna do that)
-New(Current input where the skins and lanes are stretched)
-Old(Like the osu stable build, not lazer, where skins/lanes arent stretched)

**Reply by Rekunan** [reply]
I think if this were to get added, it would be as the main option, since having to maintain multiple different input styles would be too much of a hassle.

Can I ask how skins were broken? Also, I understand ur confusion, but I'm not actually on the dev team.

**Reply by PringleKitten** [reply]
https://youtu.be/wrrEw2i8pXU?si=hx01iEhMXFFth8yt

**Reply by Rekunan** [reply]
Ahh ok, that's def something you should make a separate post for.

**Reply by PringleKitten** [reply]
Alright
